### PR TITLE
Updates for CITP Pioneer study schema

### PR DESCRIPTION
**Pull request by akohlbre**
Main changes are two new fields added to the schema (also removed overly-aggressive validation).

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] If the PR comes from a fork, trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional) and review the report posted in the comments.

For glean changes:
- [ ] Update `templates/include/glean/CHANGELOG.md`

### CODE REVIEW

**Review by acmiyaguchi — CHANGES_REQUESTED** [review]
Hi @akohlbre, could you rebase this PR onto the latest version of mozilla-pipeline-schemas. Make sure the main repo is set as a remote (e.g. `upstream`) and rebase the new patches onto the main branch. It should be something like the following:

```bash
git rebase --onto upstream/master HEAD~2
```

File: schemas/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json [hunk 1]
```diff
@@ -189,6 +168,9 @@
         }
       },
       "type": "object"
+    },
+    "WebScience.SurveyId": {
+      "type": "string"
```
Comment (acmiyaguchi) [inline schemas/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json:176]: It's good practice to document the use of this field using the `description` property, which will get inserted into the BigQuery schema.

File: schemas/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json [hunk 2]
```diff
@@ -148,21 +136,15 @@
...
                       "type": "string"
                     },
                     "classification": {
-                      "enum": [
-                        "pol news",
-                        "nonpol news",
-                        "other"
-                      ],
-                      "type": "string"
+                      "type": "integer"
```
Comment (acmiyaguchi) [inline schemas/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json:145]: This type change won't play well with automated checks in our deployment pipeline. There are two ways to change this. The first is document this field as deprecated in a description and to add a new property (e.g. `pageClassification`) which would be evolution compatible. The other is to copy the schema with the version bumped.

**Review by acmiyaguchi — CHANGES_REQUESTED** [review]

**Comment by akohlbre** [comment]
Alright --
>could you rebase this PR onto the latest version of mozilla-pipeline-schemas

Oops, sorry, I totally forgot to grab updates. Think I got the rebase to work, thanks.

>It's good practice to document the use of this field using the description property, which will get inserted into the BigQuery schema.

Added a description for the survey ID field.

>This type change won't play well with automated checks in our deployment pipeline. There are two ways to change this. The first is document this field as deprecated in a description and to add a new property (e.g. pageClassification) which would be evolution compatible. The other is to copy the schema with the version bumped.

Added the old field back and labeled it deprecated, and used your suggested name for the replacement.

Thanks!

**Review by acmiyaguchi — APPROVED** [review]

**Comment by acmiyaguchi** [comment]
Looks good to me.

**Pull request merged**